src/hnswlib/hnswalg.h

```diff
@@ -1,1353 +1,1694 @@
 #pragma once
 
+#include <assert.h>
+#include <stdlib.h>
 
-#include "visited_list_pool.h"
-#include "hnswlib.h"
-#include "adsampling.h"
 #include <atomic>
+#include <list>
 #include <random>
-#include <stdlib.h>
-#include <assert.h>
 #include <unordered_set>
-#include <list>
+
+#include "adsampling.h"
+#include "hnswlib.h"
+#include "visited_list_pool.h"
 
 using namespace std;
 using namespace adsampling;
 
 namespace hnswlib {
-    #ifdef DEEP_QUERY
-        std::vector<int>indegrees_;
-        std::vector<int>hop_cnt;
-        std::vector<float>hop_distance;
-    #endif
-    #include<random>
-    std::random_device rd;
-    std::mt19937 gen(rd());
-    std::uniform_real_distribution<float> dis(0.5, 1.0);
-
-    typedef unsigned int tableint;
-    typedef unsigned int linklistsizeint;
-
-    template<typename dist_t>
-    class HierarchicalNSW : public AlgorithmInterface<dist_t> {
-    public:
-        static const tableint max_update_element_locks = 65536;
-        HierarchicalNSW(SpaceInterface<dist_t> *s) {
-        }
-
-        HierarchicalNSW(SpaceInterface<dist_t> *s, const std::string &location, bool nmslib = false, size_t max_elements=0) {
-            loadIndex(location, s, max_elements);
+#ifdef DEEP_QUERY
+std::vector<int> indegrees_;
+std::vector<int> hop_cnt;
+std::vector<float> hop_distance;
+#endif
```

Comment:
remove all DEEP_QUERY

Reply:
I have recovered the code in your branch.

Reply:
NO, remove all the codes about DEEP_QUERY, users don't need anything about DEEP_QUERY

Reply:
OK